app/models/user.rb

```diff
@@ -15,8 +15,8 @@ class User < ApplicationRecord
     organisation_id: :organisation
   }.freeze
 
-  scope :active, -> { where(deactivated_at: nil) }
-  scope :deactivated, -> { where.not(deactivated_at: nil) }
+  scope :active, -> { where(deactivated_at: nil, anonymised_at: nil) }
+  scope :deactivated, -> { where(anonymised_at: nil).where.not(deactivated_at: nil) }
```

Comment:
I don't think I'd understood that anonymised users would disappear from both tabs before. Makes sense. Resolve this conversation if this assumption is correct 😃